packages/dappmanager/src/ethForward/ipfsProxy.ts

```diff
@@ -88,18 +88,14 @@ export function getIpfsProxyHandler<T>(
  *   to `ipfs.dappnode:8080/Qm1234/image.png`. We only need to proxy to a new hostname
  *   the hash and path information is already present in the request
  */
-function getTargetUrl(
-  proxyType: ProxyType,
-  content: Content,
-  domain: string
-): string {
+function getTargetUrl(proxyType: ProxyType, content: Content): string {
   switch (proxyType) {
     case ProxyType.ETHFORWARD:
       switch (content.location) {
         case "ipfs":
           return urlJoin(params.ETHFORWARD_IPFS_REDIRECT, "ipfs", content.hash);
         case "ipns":
-          return urlJoin(params.ETHFORWARD_IPFS_REDIRECT, "ipns", domain);
+          return urlJoin(params.ETHFORWARD_IPFS_REDIRECT, "ipns", content.hash);
```

Comment:
here i think you actually want to keep passing the domain, we don't need to resolve it to the hash